ResearchKit/Common/ORKNavigationContainerView.m

```diff
@@ -324,13 +326,18 @@ - (void)setUpConstraints {
     }
     
     {
+        CGFloat distanceFromBottom = 0.0;
+        if ([[[CEVRKTheme sharedTheme] themeName] isEqualToString:kThemeAllOfUs]) {
+            distanceFromBottom = 44 - 52;  // difference in height between theme and standard for continue button
+        }
+        
         NSLayoutConstraint *bottomConstraint = [NSLayoutConstraint constraintWithItem:_continueButton
```

Comment:
``` objective-c
NSLayoutConstraint *bottomConstraint = [[CEVRKTheme sharedTheme] navigationContainerBottomConstraint] ?: [NSLayoutConstraint constraintWithItem:_continueButton
	                                                                            attribute:NSLayoutAttributeBottom
	                                                                            relatedBy:NSLayoutRelationLessThanOrEqual
	                                                                               toItem:self
	                                                                            attribute:NSLayoutAttributeBottomMargin
	                                                                           multiplier:1.0
	                                                                             constant:0.0];
```

Reply:
I tried this by simply returning the constant to simplify some of the dependencies needed for the theme object.